docs/user/adding-glean-to-your-project.md

```diff
@@ -99,16 +99,46 @@ There are [additional parameters](android-build-configuration-options.md) that c
 #### Setting up the dependency
 
 Glean can be consumed through [Carthage](https://github.com/Carthage/Carthage), a dependency manager for macOS and iOS.
-For consuming the latest version of Glean, add the following line to your `Cartfile`::
+For consuming the latest version of Glean, add the following line to your `Cartfile`:
 
 ```
-github "mozilla/glean" "master"
+github "mozilla/glean" "{latest-version}"
 ```
 
+> **Important:** the `{latest-version}` placeholder should be replaced with the version number of the latest Glean SDK release.
+> You can find the version number on the [release page](https://github.com/mozilla/glean/releases/latest).
+
+Then check out and build the new dependency:
+
+```
+carthage bootstrap --platform iOS --color auto --cache-builds
+```
```

Comment:
If they have just checked out fresh then `bootstrap` is perfect, if they are adding this to an existing application, then it might be more efficient using `update` instead:

```bash
carthage update glean --platform iOS
```
I find this is much faster, especially when updating the Glean version to a new release.

Reply:
Technically we only support one build platform right now so the `--platform iOS` bit may be redundant.